pytissueoptics/rayscattering/opencl/__init__.py

```diff
@@ -12,9 +13,16 @@
 else:
     CONFIG = None
 
+def forceCalculationOnCPU():
+    os.environ["PYTISSUE_FORCE_CPU"] = "1"
+    # warnings.warn("You can define PYTISSUE_FORCE_CPU=1 in your profile to avoid this call.")
```

Comment:
This makes a total of 3 different endpoints to disable OpenCL, and I prefer this new apprach, so I think we should remove the parameter useHardwareAcceleration from Source.

I don't think "forceCPU" is the proper formulation. OpenCL is hardware-agnostic and will run on CPU. 
What about "PYTISSUE_DISABLE_OPENCL" and disableOpenCL() ?

Commented lines could be removed.